docs/guide-ar/start-installation.md

```diff
@@ -41,8 +41,10 @@ sudo mv composer.phar /usr/local/bin/composer
 
 <p dir="rtl">
     في هذا الدليل ، نفترض أنك قمت بتثبيت ال composer على مستوى جميع المشاريع (<a href="https://getcomposer.org/doc/00-intro.md#globally">globally</a>)  بحيث تكون أوامر ال composer متاحة لجميع المشاريع من أي مكان. أما إذا كنت تستخدم ال composer.phar لمسار محدد فقط(local directory)،  فيجب عليك ضبط  الأومر وفقًا لذلك.
+</p>
 
-إذا كان ال composer مثبتًا من قبل، فتأكد من استخدام إصدار حديث. يمكنك تحديث ال composer عن طريق تنفيذ الأمر التالي `composer self-update`
+<p dir="rtl">
+إذا كان ال composer مثبتًا من قبل، فتأكد من استخدام إصدار حديث. يمكنك تحديث ال composer عن طريق تنفيذ الأمر التالي <code>composer self-update</code>
 </p>
 
 <blockquote><p dir="rtl">
```

Comment:
btw, for blockquotes that start with `Info:`, `Warning:` etc, you can add a CSS class to make them look better.
See https://github.com/yiisoft/yii2/blob/master/docs/documentation_style_guide.md#blocks for how this works in general. But it does not apply to your case as you are using HTML, not markdown, so you would need to add the class to the element:

- `<blockquote class="info">`
- `<blockquote class="warning">`
- `<blockquote class="note">`
- `<blockquote class="tip">`

Reply:
I updated all blockqiute now by adding needed class.